aboutUS-invest.html

```diff
@@ -0,0 +1,132 @@
+    </ul>
+</section>
+<!-- content -->
+<div class="about-content">
+    <div class="Invest">
+        <div class="Invest-top">
+            <p class="Invest-title">高级客户经理</p>
+            <p class="Invest-title-bold">岗位职责要求：</p>
+            <div class="Invest-content-text">
+            <p>A、愿意把金融行业作为终身行业，个人品德端正，征信良好。 B、负责完成市场部的各项业务指标，达成个人业绩，并协助团队经理管理。 C、有很好自学能力，能不断地学习金融各项综合知识。 D、管理并维护客户关系，通过与客户沟通，及时调整客户的资金安排。 E、具备良好的户外开发能力、网络销售能力以及电话销售能力。</p>
+            <p class="Invest-title-bold">工资待遇：</p>
+            <p>
+                无责任底薪2500——4000元+个人业绩奖金+个人伯乐推荐奖+业绩提成+公司员工全面的金融培训计划+转正后五险一金+发展为团队经理的英才储备计划+发展为合伙人计划+公司股票激励计划
+            </p>
+            </div>
+        </div>
+        <div class="Invest-mid">
+            <p class="Invest-title">文案/软文</p>
+            <p class="Invest-title-bold">岗位职责要求：</p>
+            <div class="Invest-content-text">
+            <p>1. 大专以上学历，中文、新闻、文秘、广告等相关专业。</p>
+            <p>2. 2年以上相关工作经验,会基本的新媒体运营策划。</p>
+            <p>3. 优秀的文案表达能力，缜密的逻辑思维能力</p>
+            <p>4. 精通WORD、EXCEL、PPT等相关软件。 尤其具备优秀的PPT设计制作功底，熟悉PPT动态技术，能适应各类PPT美化、设计要求</p>
+            <p>5. 性格开朗，良好的沟通协调能力，做事认真负责。</p>
+            <p>6. 应聘时能提供曾做过的方案PPT、DOC文本及相关作品者优先。</p>
+            </div>
+        </div>
+        <div class="Invest-bottom">
+            <p class="Invest-title">客服</p>
+            <p class="Invest-title-bold">岗位职责要求：</p>
+            <div class="Invest-content-text">
+            <p>1.大专以上学历，年龄20-27岁；</p>
+            <p>2.1-3年公司客服岗位工作与管理经验；</p>
+            <p>3.个人形象气质好，容貌漂亮，声音甜美；</p>
+            <p>4.对本职工作认真负责，责任心强，主动性强；</p>
+            <p>5.有P2P，保险，担保，小贷行业工作经验优先。</p>
+            <p>6.优秀应届生也可。</p>
+            </div>
+        </div>
+    </div>
+</div>
+<!-- footer -->
+<footer class="clearfix">
+    <div class="footer-wrapper">
+        <div class="footer-col-link">
+            <span class="footer-col-link-title">产品</span>
+            <a href="">理财须知</a>
+            <a href="">借款须知</a>
+            <a href="">理财技巧</a>
+            <a href="">利息计算器</a>
+        </div>
+        <div class="footer-col-link">
+            <span class="footer-col-link-title">公司</span>
+            <a href="">关于我们</a>
+            <a href="">加入我们</a>
+            <a href="">合作伙伴</a>
+            <a href="">媒体报道</a>
+        </div>
+        <div class="footer-col-link footer-col-link-icon">
+            <span class="footer-col-link-title">联系我们</span>
+            <a href="">微信</a>
+            <a href="">新浪微博</a>
+            <a href="">在线客服</a>
+            <a href="">企业邮箱</a>
+        </div>
+        <div class="footer-col-qrcode">
+            扫码关注微信公众账号
+            <img src="img/QRcode.png" alt="">
+        </div>
+        <div class="footer-col-contact">
+            <p>请加官方一群：<a href=""><img src="img/group.png" alt=""></a></p>
+            <p>客服电话(工作时间: 8:30-17:30)</p>
+            <p class="footer-col-contact-phone">400-889-3878</p>
+            <p>座机电话: 0755-85250323</p>
+        </div>
+    </div>
+</footer>
```

Comment:
This part is out of date, check out the latest version ccdacbbe93c9ad0f7233734112bb0ed5f259b4cd.